lib/charms/grafana_agent_k8s/v0/grafana_agent.py

```diff
@@ -79,10 +79,15 @@ def __init__(self, *args):
 
 import logging
 import json
+import requests
```

Comment:
Can't we use `http.client`? `requests` becomes a dependency people must add in their `requirements.txt`